src/ThreadUtils_CPU.cxx

```diff
@@ -85,6 +86,9 @@ int getMaxReduceThreadsCPU()
   int nworkers = __cilkrts_get_nworkers();
   nthreads = std::max(nthreads, nworkers);
 #endif
+#if defined(RAJA_ENABLE_AGENCY)
+  nthreads = std::max(nthreads, static_cast<int>(std::thread::hardware_concurrency()));
```

Comment:
We need to be able to get the available threads if neither OpenMP or Cilk are available